plio/tests.py

```diff
@@ -774,6 +807,42 @@ def test_deleting_plio_deletes_items(self):
         response = self.client.get(f"/api/v1/items/{self.item.id}/")
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
 
+    def test_updating_item_updates_linked_plio_instance_cache(self):
```

Comment:
need to also test that updating item updates the question that is linked to the item as the item details are also returned in the question Serializer.